Spatial.tscn

```diff
@@ -183,20 +184,48 @@ __meta__ = {
 }
 
 [node name="FileDialog" type="FileDialog" parent="Control/Dialogs"]
-margin_left = 636.0
-margin_top = 174.0
-margin_right = 1307.0
-margin_bottom = 672.0
-window_title = "Open a File"
-mode = 0
-access = 1
-current_dir = "user://"
-current_path = "user://"
+visible = true
+margin_left = 312.0
+margin_top = 82.0
+margin_right = 983.0
+margin_bottom = 580.0
+window_title = "Open a Directory"
+mode = 2
+access = 2
+current_dir = "/home/steph/Desktop/Packs"
+current_path = "/home/steph/Desktop/Packs/"
+script = ExtResource( 14 )
+__meta__ = {
+"_edit_use_anchors_": false
+}
+
+[node name="WindowDialog" type="WindowDialog" parent="Control/Dialogs/FileDialog"]
+visible = true
```

Comment:
This should have been showing you a yellow warning that it should be hidden by default. (And would be hidden by default regardless of this setting)

Reply:
You are correct. Changing back to visible off.